learn/guides/asyncapi-tool.md

```diff
@@ -14,17 +14,109 @@ redirect_from:
 
 The Ballerina AsyncAPI tool will make it easy for you to start the development of an event API documented in an AsyncAPI contract in Ballerina by generating a Ballerina service and listener skeletons.
 
-> **Prerequisite:** Install the latest [Ballerina Swan Lake distribution](https://ballerina.io/downloads/).
+## Set up the prerequisites
 
-## Generate Ballerina sources from AsyncAPI contracts
+To run this tutorial, you need the following prerequisites:
 
-If you prefer the design-first approach, use an AsyncAPI definition to generate Ballerina sources using the AsyncAPI CLI command below.
+1. [Ballerina 2202.1.0 (Swan Lake)](https://ballerina.io/learn/installing-ballerina/setting-up-ballerina/) or greater
+2. A text editor
+   > **Tip:** Preferably, [Visual Studio Code](https://code.visualstudio.com/) with the [Ballerina extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) installed.
+
+## Prepare the AsyncAPI Contract
```

Comment:
```suggestion
## Prepare the AsyncAPI contract
```

Reply:
Thanks a lot for the suggestion, Resolved